index.html

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <!-- CSS only -->
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor" crossorigin="anonymous">
+<!-- JavaScript Bundle with Popper -->
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="sha384-pprn3073KE6tl6bjs2QrFaJGz5/SUsLqktiwsUTF55Jfv3qYSDhgCecCxMW52nD2" crossorigin="anonymous"></script>
+  
+<body style="background-image: url(images/survey-form.jpeg);">
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <div class="full-page">
+    <div class="container title">
+      <div class="d-flex justify-content-center">
+        <h1 id="title">freeCodeCamp Survey Form</h1>
+      </div>
+        <div class="d-flex justify-content-center">
+          <p id="description">Thank you for taking the time to help us improve the platform</p>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="container full-form col-lg-5 bg-dark w-70 p-5">
+        <form id="survey-form">
+          <div class="form-group mb-3">
+            <label id="name-label" for="name">Name</label><br>
+            <input class="form-control" id="name" type="text" name="name" placeholder="Enter your name" required>
+          </div>
+    
+          <div class="form-group mb-4">
```

Comment:
I will suggest you close this space here too and any other such spaces

Reply:
Kindly include an explanation, a "why", for each requested change. This will help them see the reasons with you. Some reasons might seem obvious to you but not to the author of the code.